src/main/java/com/trzewik/spring/domain/game/Result.java

```diff
@@ -26,4 +26,10 @@ public Exception(String msg) {
             super(msg);
         }
     }
+
+    public enum GameResult {
```

Comment:
IMHO - more relevant name for this enum will be `PlayerResult` or `PlayerGameResult`. Game can't WIN or LOSS - only player has this possibility

Reply:
Done